docs/install-binary.md

```diff
@@ -1 +1,152 @@
 # Binary install
+
+To start developing with Nakama, you’ll first need to install it on your development machine. It’s straightforward and takes just a few minutes. In this guide we’re going to focus on installing your Nakama development instance using the binary executable.
+
+!!! summary "Recommended Approach"
+    [Docker](install-docker-quickstart.md) is the quickest way to download and get started with Nakama for development purposes. For production settings, we recommend that you install Nakama as a binary to ensure all system resources are available to Nakama.
+
+## Requirement
+There are a few things you’ll need to know or have to hand before you install Nakama:
+
+- Operating system: Windows 10 64-bit or above, MacOS 10.9 or above, or a modern Linux.
+- Architecture: X86_64 (64bit) processor architecture
+- Dependent software: Nakama relies on [CockroachDB](https://cockroachlabs.com).
+
+## Install CockroachDB
+
+Nakama relies on CockroachDB as the main and only database. Nakama uses CockroachDB to store server configuration, user data, chat messages and more persistent data.
+
+You'll first need to install CockroachDB. Follow [this guide](https://www.cockroachlabs.com/docs/stable/install-cockroachdb.html) to install CockroachDB on your machine before installing Nakama.
+
+!!! note "CockroachDB Alternative"
+    Nakama unofficially supports [PostgreSQL](https://www.postgresql.org/download/) for environments where CockroachDB is not available.
+
+    For production settings, Nakama should be used with CockroachDB as queries are optimised due to the way CockroachDB stores data on the disk.
+
+## Install Nakama on MacOS
+
+You’ll need to be running MacOS 10.9 (Mavericks) to run Nakama.
+
+### Homebrew
```

Comment:
We can leave it here, and if anyone wants to try it, they can message us. 

Most of the focus is on Docker anyways :)